libraries/lib-stretching-sequence/StretchingSequence.cpp

```diff
@@ -183,14 +183,15 @@ bool StretchingSequence::MutableGet(
          t,
          backwards ? PlaybackDirection::backward : PlaybackDirection::forward);
    }
-   return GetNext(reinterpret_cast<float *const *>(buffers), nBuffers, len);
+   return GetNext(reinterpret_cast<float* const*>(buffers), nBuffers, len);
 }
 
 std::shared_ptr<StretchingSequence> StretchingSequence::Create(
    const PlayableSequence& sequence, const ClipHolders& clips)
 {
+   const int sampleRate = sequence.GetRate();
    return std::make_shared<StretchingSequence>(
-      sequence, sequence.GetRate(), sequence.NChannels(),
+      sequence, sampleRate, sequence.NChannels(),
       std::make_unique<AudioSegmentFactory>(
-         sequence.GetRate(), sequence.NChannels(), clips));
```

Comment:
What compilation warning was fixed by the first commit?

Reply:
loss of precision warning (size_t vs int or the other way round)